src/main/java/gov/usds/case_issues/services/PageTranslationService.java

```diff
@@ -0,0 +1,13 @@
+package gov.usds.case_issues.services;
+
+import gov.usds.case_issues.services.CaseListService.CaseGroupInfo;
```

Comment:
Hoisted from a self-comment on #94: this is a terrible smell right here—the interface refers to classes that are inner classes of the (only) implementation class.

I hate having four different `model` packages, but these probably should go into a `services.model` subpackage.